tests/rustdoc/extern/duplicate-reexports-section-150211.rs

```diff
@@ -0,0 +1,19 @@
+//@ aux-build:pub-extern-crate.rs
+
+// Regression test for issue #150211.
+// When a module has both `pub extern crate` and `pub use` items,
+// they should both appear under a single "Re-exports" section,
+// not two separate sections.
+// https://github.com/rust-lang/rust/issues/150211
```

Comment:
```suggestion
// Regression test for issue <https://github.com/rust-lang/rust/issues/150211>.
// When a module has both `pub extern crate` and `pub use` items,
// they should both appear under a single "Re-exports" section,
// not two separate sections.
```

I don't think there's a need to duplicate issue numbers.